assets/component-volume-pricing.css

```diff
@@ -0,0 +1,42 @@
+volume-pricing {
+  display: block;
+  margin-top: 2rem;
+}
+
+volume-pricing li:nth-child(odd) {
+  background: rgba(var(--color-foreground), 0.04);
+}
+
+volume-pricing ul {
+  margin-top: 1.2rem;
+}
+
+volume-pricing li {
+  font-size: 1.4rem;
+  letter-spacing: 0.06rem;
+  padding: 0.8rem 1.2rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.volume-pricing-note {
+  margin-top: -2.6rem;
```

Comment:
Is there a reason we use negative margin here? I feel we try to avoid this unless there's no other option.

Reply:
@KaichenWang We want to make the note closer to the price as per @scottgmeadows 's feedback: https://github.com/Shopify/dawn/pull/2711#issuecomment-1597274549

Because the spacing per blocks is automatic: https://screenshot.click/27-39-2cd65-plsgv.png